src/m365/commands/docs.spec.ts

```diff
@@ -0,0 +1,82 @@
+import Command from '../../Command';
+import commands from './commands';
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import { Logger } from '../../cli/Logger';
+import { telemetry } from '../../telemetry';
+import { pid } from '../../utils/pid';
+import { session } from '../../utils/session';
+import { sinonUtil } from '../../utils/sinonUtil';
+import { Cli } from '../../cli/Cli';
+const packageJSON = require('../../../package.json');
+
+const command: Command = require('./docs');
+
+describe(commands.DOCS, () => {
+  let log: any[];
+  let logger: Logger;
+  let cli: Cli;
+  let loggerLogSpy: sinon.SinonSpy;
+  let getSettingWithDefaultValueStub: sinon.SinonStub;
+  let openBrowserSpy: sinon.SinonSpy;
+
+  before(() => {
+    sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
+    sinon.stub(pid, 'getProcessName').callsFake(() => '');
+    sinon.stub(session, 'getId').callsFake(() => '');
+    (command as any).open = () => { };
+    openBrowserSpy = sinon.spy(command as any, 'openBrowser');
```

Comment:
If we go with the implementation I proposed, I think we should convert this spy to a stub that returns a resolved promise. Then it should be fine to remove the line above as well.